Frontend/ui-driver/src/Screens/TripDetailsScreen/View.purs

```diff
@@ -286,12 +286,10 @@ tripDataView push state =
         [ height WRAP_CONTENT
         , width MATCH_PARENT
         , orientation HORIZONTAL
-        , gravity CENTER_VERTICAL
         ][ linearLayout
             [ height WRAP_CONTENT
-            , width WRAP_CONTENT
+            , width $ V $ (EHC.screenWidth unit) / 2
```

Comment:
Why are we hardcoding view width on many places ?

Reply:
its not hard coding. we want to show two views like grid. so taken half width